.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: "CI"
+
+on:
+    pull_request:
+    push:
+        branches:
+            - '3.x'
+
+env:
+    fail-fast: true
+    SYMFONY_PHPUNIT_DISABLE_RESULT_CACHE: 1
+
+jobs:
+    tests:
+        name: "PHP ${{ matrix.php-version }}"
+
+        runs-on: 'ubuntu-latest'
+
+        continue-on-error: ${{ matrix.allow-failures }}
+
+        strategy:
+            matrix:
+                php-version:
+                    - '7.2.5'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                allow-failures: [false]
```

Comment:
shouldn't this be `fail-fast: false` instead ?

Reply:
I use this variable above on `continue-on-error`, `fail-fast: true` is globally on (see line 9)

Reply:
IMO, we should not have `fail-fast: true` on the matrix. If the PHP 7.2 job fails, I don't want to make it cancel the jobs for other PHP versions. It would hide results about which PHP versions are impacted by a failure.

Reply:
Will remove it then, but do you agree on executing extension tests only if the tests for the main repo pass?